esmvaltool/diag_scripts/cos22esd/hotspot_plotter.py

```diff
@@ -194,8 +194,7 @@ def _hotspot_fields_plot_panels(self, results_dict, fig, keys, style):
 
     @staticmethod
     def _add_axes_attributes(axes, path_ext, plotextend):
-        axes.set_boundary(path_ext, transform=ccrs.PlateCarree(),
-                          use_as_clip_path=True)
+        axes.set_boundary(path_ext, transform=ccrs.PlateCarree())
```

Comment:
This modification does not affect the results or the figure style. It looks good from my side